app/assets/javascripts/autocise_textarea.js

```diff
@@ -0,0 +1,53 @@
+// Get the max height and apply
+// to all textareas when page loads/reloads
+
+$(document).ready(function() {
+  var numberArray = [];
+  $('textarea').each(function(textarea) {
+    $(this).height( $(this)[0].scrollHeight );
+    numberArray.push($(this).innerHeight());
+    var max = Math.max(...numberArray);
+    $(this).innerHeight(max);
+  });
+});
+
+// Use an event to listen to all input events and then
+// filter out ones that are not on a textarea
+// if the element is a textarea we will call a
+// autoExpandTextarea function and pass in
+// the element as an argument using event.target.
+document.addEventListener('input', function(event) {
+  if (event.target.tagName.toLowerCase() !== 'textarea') return;
```

Comment:
If you only want this behavior on a textarea input, then I would attach the event listener _only_ on that input. Additionally, since we're using jQuery on this site, it's best to create event handlers consistently.

The equivalent would look like:
```
$('textarea').change(function(event) {...});
```

Reply:
its javascript - much better performance than jquery

Reply:
jquery is javascript anyway

Reply:
I think also this will perform better on IE